pyuploadcare/api_resources.py

```diff
@@ -625,13 +621,8 @@ class BaseApiList(object):
     filters = []
 
     def __init__(self, **kwargs):
-        # Only explicit kwargs are allowed.
-
-        if kwargs.get('since') is not None and kwargs.get('until') is not None:
-            raise TypeError('Only one of since and until arguments is allowed.')
-
-        self.since = kwargs.pop('since', None)
-        self.until = kwargs.pop('until', None)
+        self.starting_point = kwargs.pop('starting_point', None)
+        self.ordering = kwargs.pop('ordering', None)
```

Comment:
Don't know why all arguments are in `kwargs`. Could you try to make regular arguments with default values?